src/Http/Http.Extensions/test/RequestDelegateGenerator/CompileTimeCreationTests.AsParameters.cs

```diff
@@ -39,16 +39,16 @@ public static object[][] BadArgumentListActions
         get
         {
             static string GetAbstractTypeError(Type type)
-                => $"The abstract type '{TypeNameHelper.GetTypeDisplayName(type, fullName: false)}' is not supported.";
+                => $"The abstract type '{TypeNameHelper.GetTypeDisplayName(type, fullName: false)}' is not supported. For more information, please see https://aka.ms/aspnet/rdg-known-issues";
 
             static string GetMultipleContructorsError(Type type)
-                => $"Only a single public parameterized constructor is allowed for type '{TypeNameHelper.GetTypeDisplayName(type, fullName: false)}'.";
+                => $"Only a single public parameterized constructor is allowed for type '{TypeNameHelper.GetTypeDisplayName(type, fullName: false)}'. For more information, please see https://aka.ms/aspnet/rdg-known-issues";
```

Comment:
Will issues/info about each of these things be added to that known issues issue? Right now, there's no further information about this limitation at that link.

I guess the meta-question is, if the error is descriptive enough about what the problem is, is there value in adding the link?

Reply:
> Will issues/info about each of these things be added to that known issues issue? Right now, there's no further information about this limitation at that link.

They'll be added to a diagnostics info page in the docs similar to what we have in [for analyzer diagnostics like this one](https://learn.microsoft.com/en-us/aspnet/core/diagnostics/asp0014?view=aspnetcore-7.0).

The aka.ms link currently directs to the old known issues GitHub page but the page on docs.microsoft.com will enumerate through more of the warnings.

Reply:
Gotcha, thanks.